content/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-files.md

```diff
@@ -29,9 +29,10 @@ Each entry defines a lookup name - a logical name for the resource - and a colle
 * AccountId - Used to override the global account id attribute, if this resource is deployed in a different AWS account.
 
 **Global attributes**
-The mapping file also supports three **required** global attributes:
 
-* AccountId - The default AWS account containing resources in this mapping file.
+The mapping file also supports one _optional_ and two **required** global attributes:
+
+* AccountId (optional) - The default AWS account that is used to search for AWS resources referenced in your O3DE application code.
```

Comment:
nit: The default  aws account id to use when no account id is explicitly referenced ?